APIManager.js

```diff
@@ -0,0 +1,74 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+  constructor() {
+    this.myData = {
+      quote: {},
+      bacon: {},
+      mainuser: {},
+      pokemone: {},
+      friends: []
+  }
+  
+}
+
+      getData() {
+        this.getBacon()
+        this.getQuote()
+        this.getUsers()
+        this.getPokemon()
+    }
+
+  getQuote() {
+    $.ajax({
+      method: "GET",
+      url: "https://api.kanye.rest?format=text",
+      success: (data) => {
+        this.myData.quote = data;
+      },
+      error: function (xhr, text, error) {
+        alert("not working");
```

Comment:
it would also be good to `console.log(error)` then you know exactly what happened